src/conversions.rs

```diff
@@ -481,12 +501,15 @@ impl<T: FromJSValConvertible> FromJSValConvertible for Option<T> {
     unsafe fn from_jsval(cx: *mut JSContext,
                          value: HandleValue,
                          option: T::Config)
-                         -> Result<Option<T>, ()> {
+                         -> Result<ConversionResult<Option<T>>, ()> {
         if value.get().is_null_or_undefined() {
-            Ok(None)
+            Ok(ConversionResult::Failure(Cow::Owned("Value is null or undefined.".to_owned())))
```

Comment:
What's the point of returning a `Cow` if we only return owned values? Can't this be `Cow::Borrowed("Value is null or undefined")`?